content/hardware/02.hero/boards/uno-r4-wifi/datasheet/datasheet.md

```diff
@@ -474,3 +474,4 @@ Hereby, Arduino S.r.l. declares that this product is in compliance with essentia
 | ---------- | ------------ | ------------------ |
 | 08/06/2023 | 1            | First Release      |
 | 30/06/2023 | 2            | Update Pinout File |
+| 25/07/2023 | 3            | Update Pin Table   |
```

Comment:
We are actually following the format:

Most recent first
European date system: dd/mm/yyyy

So

```suggestion
| 25/07/2023 | 3            | Update Pin Table   |
| 30/06/2023 | 2            | Update Pinout File |
| 08/06/2023 | 1            | First Release      |
```